### ISSUE TITLE
Description of InternalXmlHelper Class doesn't mention .NET Core

### ISSUE THREAD

**Issue by TymurGubayev** [dotnet/dotnet-api-docs#7268]
On https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.compilerservices.internalxmlhelper?view=net-6.0 only .NET Framework 4.0-4.8 is mentioned
![image](https://user-images.githubusercontent.com/348150/135882254-af1a2ed2-7ee8-478c-8c94-21f90bf31bca.png)

But the class is present at least in Microsoft.VisualBasic.dll Version 10.1.0.0 / 5.0.20.52001 (i.e. .NET 5.0)

**Comment by safern** [comment]
I don't think this API is exposed in .NET 6 as an available API to compile against. It it not exposed on any reference assembly in .NET Core, the shims that @gewarren is referring to are to help net472 types unify when running on .NET Core but that is partially supported and we don't promise it will work, this is in the case there is a NuGet package that targets netfx and doesn't provide a .NET Core asset. We will emit a warning at restore, but that is a warning people can suppress and then it will most of the time work fine because of these facades.

However, these facades don't have anything to do with which APIs are supported on .NET Core or .NET5+, meaning not every type that is shimmed is exposed at compile time for .NET Core or .NET5+, this is one of those cases. 

You can for example validate that by creating a new console app for VB that targets .NET 6 in this case and try to reference `InternalXmlHelper`, the type will not be found on any namespace.

![image](https://user-images.githubusercontent.com/22899328/137971861-38c44745-3492-4b84-896c-ce04d0d44ce0.png)

**Comment by gewarren** [comment]
Thanks for the clarifications. I'll close this issue now.
